maintainers/maintainer-list.nix

```diff
@@ -3341,6 +3341,12 @@
     githubId = 1882000;
     name = "Guillaume Desforges";
   };
+  guillaumecherel = {
+    email = "guillaume.cherel@cosimus.com";
+    github = "guillaumecherel";
+    githubId = 6168820;
+    name = "Guillaume Chérel";
+  };
   guillaumekoenig = {
```

Comment:
Please make this into a separate commit:

```
maintainers: add <name>
```

Reply:
Nitpick: commit messages should not end with a dot.